packages/near-membrane-dom/src/browser-realm.ts

```diff
@@ -125,6 +126,9 @@ export default function createVirtualEnvironment(
         redConnector,
         distortionCallback,
     });
+    env.link('window');
```

Comment:
for a browser, we first link `globalThis.window` references, then the intrinsics to preserve identities across the membrane, then the unforgeables, and finally we tame the DOM apis.